<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AIAssistDialog</class>
 <widget class="QDialog" name="AIAssistDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AI Assist</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="systemPromptLabel">
     <property name="text">
      <string>Quick Action:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="systemPromptCombo">
     <item>
      <property name="text">
       <string>Custom</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="promptLabel">
     <property name="text">
      <string>Prompt:</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="promptLayout">
     <item>
      <widget class="QLineEdit" name="promptEdit">
       <property name="placeholderText">
        <string>Enter your instruction (e.g., "Summarize this", "Fix grammar", "Expand this idea")</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="processButton">
       <property name="text">
        <string>Process</string>
       </property>
       <property name="toolTip">
        <string>Send prompt and content to AI provider (Ctrl+Return)</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="contentLabel">
     <property name="text">
      <string>Content:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPlainTextEdit" name="contentEdit">
     <property name="placeholderText">
      <string>Enter or paste content here...</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="historyLayout">
     <item>
      <widget class="QPushButton" name="prevButton">
       <property name="text">
        <string>◄ Previous</string>
       </property>
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="historyLabel">
       <property name="text">
        <string>1/1</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="nextButton">
       <property name="text">
        <string>Next ►</string>
       </property>
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="buttonLayout">
     <item>
      <spacer name="buttonSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="applyButton">
       <property name="text">
        <string>Insert</string>
       </property>
       <property name="toolTip">
        <string>Apply content to editor</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="copyButton">
       <property name="text">
        <string>Copy to Clipboard</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>AIAssistDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>540</x>
     <y>470</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>249</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
